source/fuzz/fuzzer_pass_add_vector_shuffle_instructions.cpp

```diff
@@ -73,6 +73,12 @@ void FuzzerPassAddVectorShuffleInstructions::Apply() {
                     return false;
                   }
 
+                  if (GetTransformationContext()
+                          ->GetFactManager()
+                          ->IdIsIrrelevant(instruction->result_id())) {
+                    return false;
+                  }
```

Comment:
Hmm, perhaps we *should* allow vector shuffles on irrelevant vectors, but just suppress the deduction of synonym facts in the transformation in cases where the vector involved is irrelevant?

Reply:
Fixed in 35f5730.